guides/main_concepts/search.md

```diff
@@ -1,21 +1,21 @@
 # Search
 
-### Finding documents
+## Finding documents
 
 When the query input is received, MeiliSearch is building a more complex query taking into account *typos*, n-grams, and *synonyms* if configured.
 
 - _Typos_ - For example, if the query string is `botman`, MeiliSearch will return documents containing `batman`. [Read more about the typo rules](/guides/advanced_guides/typotolerance.md).
-- _N-grams_ - MeiliSearch is set to merge multi-words query into a single word. Ex: Searching for `bat mobile` will returns documents containing `batmobile`. Each words of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`.
+- _N-grams_ - MeiliSearch is set to merge multi-words query into a single word. Ex: Searching for `bat mobile` will returns documents containing `batmobile`. Each words of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`. [Read more about Concatenate and Split Queries](/guides/advanced_guides/concat.md)
```

Comment:
```suggestion
- _N-grams_ - MeiliSearch is set to merge multi-words query into a single word. Ex: Searching for `bat mobile` will returns documents containing `batmobile`. Each word of the query will also be split in many ways so MeiliSearch can returns documents containing `new york` when querying for `newyork`. [Read more about Concatenate and Split Queries](/guides/advanced_guides/concat.md)
```

Reply:
"multi-words query"? (sorry, I know it was there before your change, but I don't understand what it means. Especially because Grammarly would like "multi-words queries" instead.)

Reply:
Multi words query is when you do a search with more than one word: 
ex : "Hello world" > two words, "Mask" > one word
Thus "Hello world" is a multi-word query.

Reply:
Ok in this case, I will do a suggestion because I haven't understood the sentence before